src/libs/curated_app_libs.py

```diff
@@ -65,14 +67,12 @@ def pre_actions(test_config_dict):
     end_key = test_config_dict.get("end_test")
     if os.path.isdir(CURATED_APPS_PATH + "/test_config"):
         shutil.rmtree(CURATED_APPS_PATH + "/test_config")
-    else:
-        os.mkdir(CURATED_APPS_PATH+"/test_config")
-    utils.run_subprocess("cp -rf test_config/* {}".format(CURATED_APPS_PATH+"/test_config"))
+    shutil.copytree("test_config", CURATED_APPS_PATH + "/test_config")
 
     pre_actions_for_verifier_image(test_config_dict, end_key)
 
-    input_ord_list = ['signing_key_path', 'runtime_args', 'runtime_variables', 'runtime_variable_list', 'attestation', 
```

Comment:
don't we need the ssl_path also?

Reply:
curated app wont ask for ssl_path.. User has to place the crt files in ssl folder manually.